nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/src/test/java/org/apache/nifi/processors/solr/util/MockRecordParser.java

```diff
@@ -0,0 +1,105 @@
+/*

```

Comment:
See `TestPutHBaseRecord` for an example of how to use `org.apache.nifi.serialization.record.MockRecordParser` which should be able to replace this class.

Reply:
`PutMongoRecordIT.testInsertNestedRecords` is another good example.

Reply:
Sure, I'll look into these tests.